Jenkinsfile

```diff
@@ -14,7 +14,7 @@ node {
   
   stage ("Deploy"){
     NAMESPACE = k8s.getNamespaceFromBranch(env.BRANCH_NAME)
-    if (env.BRANCH_NAME in ['rc']) {
+    if (env.BRANCH_NAME in ['rc', 'master']) {
```

Comment:
This is an excellent place to try out dev with no staging.

Reply:
Patched to internal-master in latest push